app/forms/aboutyou/TaxAvoidanceSchemesFormProvider.scala

```diff
@@ -33,7 +34,7 @@ class TaxAvoidanceSchemesFormProvider @Inject() extends Mappings {
         "taxAvoidanceSchemes.error.required"
       }
     Form(
-      "value" -> set(enumerable[TaxAvoidanceSchemes](error)).verifying(nonEmptySet(error))
+      "value" -> set(enumerable[TaxAvoidanceSchemes](error)).verifying(nonEmptySet(error), exclusiveItemInSet(error, NoAvoidance.toString))
```

Comment:
Related to the other comment, should this be using `firstError()` around the constraints so we're not doing this in the view models?

For example: `verifying(firstError(constraint1, constraint2))` where the form is a `value -> list(enumerable[T])`